### ISSUE TITLE
No Hover Styles and Requiring Double Click to Click

### ISSUE THREAD

**Issue by mutewinter** [HackPlan/quark-shell-mac#37]
Sometimes when the app is opened the window doesn't seem to be acknowledging the focus. Hover styles are not shown for elements within the web view and two clicks are required to activate anything. The window also won't dismiss when clicking outside.

**Comment by xhacker** [comment]
I can't reproduce the issue, can you give me some steps? Thanks!

**Comment by mutewinter** [comment]
Hmm, perhaps it happened because I resized the window in interface builder. I'll see if I can repro from a fresh download.

**Comment by mutewinter** [comment]
Got a repro! I just cloned [master](https://github.com/HackPlan/quark-shell-mac/commit/e781bd390fa23f49b7a235e8814f1906cf922834), setup Cocoa Pods, and made this `index.html`:

```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8">
    <title></title>
  </head>
  <body>
    <script>
      window.location.href = 'https://github.com'
    </script>
  </body>
</html>
```

To reproduce hover styles / need to double click:

1. click the menu bar icon
1. command tab to another app
1. click the menu bar icon again
1. now all links will not show hover state and clicks take two clicks to activate

[Here's a video](https://dl.dropboxusercontent.com/u/795597/quark-shell-mac-double-click-bug.mov) of how the app behaves after the bug occurs.

Running 10.10.2 and Safari 8.0.4.

**Comment by mutewinter** [comment]
Fixed by 10.10.3!

**Comment by xhacker** [comment]
Awesome! Thanks for letting me know :)
